packages/@tinacms/schema-tools/src/types/index.ts

```diff
@@ -1,19 +1,705 @@
 /**
-
 Copyright 2021 Forestry.io Holdings, Inc.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-
 */
+// Easier to copy this from hardcoded react file than keeping it as a dependency
+import type React from './react'
```

Comment:
Just copied the types from react and hardcoded them, I think our needs are simple enough for that to work